docker-compose-esignet/README.md

```diff
@@ -0,0 +1,61 @@
+## Overview
+
+This is the docker-compose setup to run esignet UI and esignet-service with mock identity system. This is not for production use.
+
+## What is in the docker-compose setup folder?
+
+1. "app" folder holds the Dockerfile required to build custom artifactory-server. This artifactory server will host all the files under app/static folder.
+All the i18n bundles, dummy softhsm conf, signin-with-esignet button plugin files are served from this server. 
+2. "config" folder holds the esignet and mock-identity system properties file.
+3. "docker-compose.yml" file with esignet and mock-identity-system setup with other required services
+4. "init.sql" comprises DDL and DMLs required by esignet and mock-identity-system.
+5. "loader_path" this is esignet mount volume from where all the runtime dependencies are loaded to classpath. If any new esignet plugins to be tested
+should be placed in this folder and respective plugin configuration should be updated in config/esignet-default.properties.
+
+```Note: Refer https://docs.esignet.io/integration to know how to create custom plugins to integrate.```
+
+## How to run this setup?
+
+1. Start the docker-compose file
+
+> docker-compose up
+
+2. Download the postman script from [here](https://github.com/mosip/esignet/blob/master/docs/postman-collections/esignet-OIDC-flow-with-mock.postman_collection.json)
+and its environment from [here](https://github.com/mosip/esignet/blob/master/docs/postman-collections/esignet-OIDC-flow-with-mock.postman_environment.json)
+
+3. Import the downloaded collection and environment into postman.
+
+4. To Create a Mock identity, run the below request from the postman collection "Mock-Identity-System" folder
+   * Create Mock Identity
+
+5. To create an OIDC/OAuth client, run the below request from the postman collection "OIDC Client mgmt" folder
+   * Get CSRF token
+   * Create OIDC Client
```

Comment:
Can we simplify this using a shell script for the same ?

Reply:
using postman collection for now to create client